engine/runtime/src/main/java/org/enso/interpreter/runtime/callable/atom/Atom.java

```diff
@@ -86,13 +90,25 @@ private void toString(StringBuilder builder, boolean shouldParen, int depth) {
    */
   @Override
   public String toString() {
-    return toString(10);
+    return toString(null, 10, null, null);
   }
 
   @CompilerDirectives.TruffleBoundary
-  private String toString(int depth) {
+  private String toString(String prefix, int depth, String suffix, Object obj) {
     StringBuilder sb = new StringBuilder();
+    if (prefix != null) {
+      sb.append(prefix);
+    }
     toString(sb, false, depth);
+    if (suffix != null) {
+      sb.append(suffix);
+    }
+    if (obj != null) {
+      var errorMessage = InteropLibrary.getUncached().toDisplayString(obj);
+      if (errorMessage != null) {
+        sb.append(errorMessage);
+      }
```

Comment:
```suggestion
      if (errorMessage != null) {
        sb.append(errorMessage);
      } else {
        sb.append("Unknown error.");
      }
```

As our messages all say stuff along the lines of `Error in method `to_text` of <foo>:` -so we have this colon suggesting something is to follow, I think in the `null` case we still should have something after that colon, so that the message does not look cut off halfway through.

Reply:
Printing `Nothing` in [4e55305](https://github.com/enso-org/enso/pull/4034/commits/4e55305e688d69a842ebffde063694bba22c4fcf)